src/client/dfuse/src/SConscript

```diff
@@ -0,0 +1,224 @@
+# Copyright (C) 2016-2019 Intel Corporation
```

Comment:
dfuse/src?

Reply:
Minor rework of the scons files is needed here to remove the intermediate directory.

Presumably we'd want to move test code (if we're keeping it) and internal header files elsewhere in the DAOS tree at the same time.